UPDATING.md

```diff
@@ -57,6 +57,31 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### Chain and cross_downstream moved from helpers to BaseOperator
+
+The chain and cross_downstream methods are now moved to airflow.models.baseoperator module from
+airflow.utils.helpers module where they belong.
```

Comment:
```suggestion
`airflow.utils.helpers` module where they belong.
```

Reply:
Yep. I will also update the description slightly.